python/pylibvw.cc

```diff
@@ -1102,6 +1102,19 @@ float get_total_sum_feat_sq(example_ptr ec) { return ec->get_total_sum_feat_sq()
 double get_sum_loss(vw_ptr vw) { return vw->sd->sum_loss; }
 double get_holdout_sum_loss(vw_ptr vw) { return vw->sd->holdout_sum_loss; }
 double get_weighted_examples(vw_ptr vw) { return vw->sd->weighted_examples(); }
+float get_explore_eval_weighted_update_count(vw_ptr vw)
+{
+  return vw->sd->reduction_stats.count("explore_eval_weighted_update_count")
```

Comment:
If it doesn't exist then perhaps we should return `None`?

Reply:
mmm yeah probably